lessons/PlottinginPythonwithMatplotlib/hwk3.1.py

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8 -*-
+from numpy import float32
+from numpy import linspace
+from numpy import polyfit
+from numpy import polyval
+import matplotlib.pyplot as plt
+
+#Read in data from csv
+f=open('data.csv','r')
+line=f.readlines()
+
+#Empty array for data
+FN=[]
+EFN=[]
+#This loop goes through every line, strips new line character and then splits the data on ,.  It will then save data into the arrays
+for l in line:
```

Comment:
Wouldn't it be easier for people to see which modules are loaded and what
is being called?


On Wed, Aug 5, 2015 at 2:59 PM, Elliott Sales de Andrade <
notifications@github.com> wrote:

> In lessons/PlottinginPythonwithMatplotlib/hwk3.1.py
> <https://github.com/UofTCoders/studyGroup/pull/33#discussion_r36340432>:
>
> > +# -*- coding: utf-8 -*-
> > +from numpy import float32
> > +from numpy import linspace
> > +from numpy import polyfit
> > +from numpy import polyval
> > +import matplotlib.pyplot as plt
> > +
> > +#Read in data from csv
> > +f=open('data.csv','r')
> > +line=f.readlines()
> > +
> > +#Empty array for data
> > +FN=[]
> > +EFN=[]
> > +#This loop goes through every line, strips new line character and then splits the data on ,.  It will then save data into the arrays
> > +for l in line:
>
> This is very long. Why not use np.genfromtxt?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/UofTCoders/studyGroup/pull/33/files#r36340432>.
>